### PR TITLE
Add first headline feature

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Audits listing performance tweaks
+
+Loading audits could get very slow and resource consuming in case the audits were scoped to specific organization or location. With this release, the SQL queries were optimized to load the correct audit records faster. ([#30053](https://projects.theforeman.org/issues/30053))
```
Comment (ekohl) [inline _includes/manuals/nightly/1.2_release_notes.md:9]:
```suggestion
Loading audits could be very slow and resource consuming when the audits were scoped to a specific organization or location. With this release, the SQL queries were optimized to load the correct audit records faster. ([#30053](https://projects.theforeman.org/issues/30053))
```

**Comment by tbrisker** [comment]
We have branched the manual for 2.5, please update this PR accordingly.

**Comment by lzap** [comment]
This workflow is abysmal, I already suggested tracking headline feature and upgrade notes in a thread on discourse until there is the right time to pull it. 🤷🏻

**Comment by ares** [comment]
closing in favor of where I implemented the suggestions https://github.com/theforeman/theforeman.org/pull/1842

**Pull request closed without merging**